documentation/reference/query-language-reference.html

```diff
@@ -805,6 +805,65 @@ <h2 id="where">where</h2>
     </table>
   </td></tr>
 
+  <tr id="geoLocation"><th>geoLocation</th><td>
```

Comment:
This is already documented: https://docs.vespa.ai/documentation/reference/query-language-reference.html#geoLocation, and duplicates the row above.